examples/cra/src/components/getTrades/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { FormEvent, useCallback, useEffect, useState } from 'react'
+import '../../pageStyles.css'
+import { OrderBookApi, Trade } from '@cowprotocol/cow-sdk'
+import { useWeb3Info } from '../../hooks/useWeb3Info'
+import { parseFormData } from '../../utils'
+
+const orderBookApi = new OrderBookApi()
```

Comment:
Would it make sense to initialize this only once in some file and then import the same instance?

Reply:
I would like to keep it as it to provide an opportunity just copy-paste the code